cv_next/server/api/cvs.ts

```diff
@@ -1,4 +1,5 @@
-import "server-only";
+"use server";
+//import "server-only";
```

Comment:
Again, don't leave dangling comments.

@Aradon267 do we have a git hook for this? Preferably pre-commit

Reply:
Also, why remove the import server only?

Reply:
I remember that I added pre-commit, and also an eslint rule to warn at console.log, will look into it